distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -700,6 +700,36 @@ type Decimal
     signum : Integer
     signum self -> Integer = self.big_decimal.signum
 
+    ## GROUP Math
+       ICON sigma
+       Returns the smaller value of `self` and `that`.
+
+       Arguments:
+       - that: The number to compare `self` against.
+
+       > Example
+         Find the minimum of 12 and 13.
+
+             Decimal.new "12" . min (Decimal.new "13")
+             # => Decimal.new "12"
+    min : Number -> Number
+    min self that = if self < that then self else that
+
+    ## GROUP Math
+       ICON math
+       Returns the larger value of `self` and `that`.
+
+       Arguments:
+       - that: The number to compare `self` against.
+
+       > Example
+         Find the maximum of 12 and 13.
+
+             Decimal.new "12" . max (Decimal.new "13")
+             # => Decimal.new "13"
+    max : Number -> Number
+    max self that = if self > that then self else that
```

Comment:
```suggestion
    max self (that : Number) = if self > that then self else that
```

It feels like we should also have the checked type ascription here - this should probably make for better widgets in the GUI (at least in future) and will aid type checking.

Reply:
Done -- actually changed to `Decimal`; `Number` was wrong.